src/System.Runtime.Extensions/tests/System/IO/PathTests.Windows.netcoreapp.cs

```diff
@@ -0,0 +1,37 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using Xunit;
+
+namespace System.IO.Tests
+{
+    public partial class PathTests_Windows : PathTestsBase
```

Comment:
If this refactoring is perfmanent, the name of the file should be `PathTests_Windows` to follow the naming conventions.

Reply:
(Similar for other places.)